src/core/Akka.Persistence.TestKit/JournalRecoveryBehavior.cs

```diff
@@ -0,0 +1,194 @@
+//-----------------------------------------------------------------------
+// <copyright file="JournalRecoveryBehavior.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2019 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2019 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Akka.Persistence.TestKit
+{
+    using System;
+    using System.Threading.Tasks;
+
+    public class JournalRecoveryBehavior
+    {
+        internal JournalRecoveryBehavior(IJournalBehaviorSetter setter)
+        {
+            this.Setter = setter;
+        }
+
+        protected readonly IJournalBehaviorSetter Setter;
+
+        public Task SetInterceptorAsync(IJournalInterceptor interceptor) => Setter.SetInterceptorAsync(interceptor);
```

Comment:
`SetInterceptorAsync` now is `public` so that it is possible to have custom interceptors.

Reply:
Nice idea